internal/service/domain/treecluster/treecluster.go

```diff
@@ -232,7 +232,15 @@ func (s *TreeClusterService) Ready() bool {
 // otherwise the center point of the tree cluster cannot be set
 func (s *TreeClusterService) updateTreeClusterPosition(ctx context.Context, id int32) error {
 	log := logger.GetLogger(ctx)
-	err := s.treeClusterRepo.Update(ctx, id, func(tc *domain.TreeCluster) (bool, error) {
+	wateringStatus, err := s.getWateringStatusOfTreeCluster(ctx, id)
+	if err != nil {
+		slog.Error("could not update watering status", "error", err)
```

Comment:
Please change to `log`
```suggestion
		log.Error("could not update watering status", "error", err)
```